crates/beerus-rpc/src/beerus_rpc_server.rs

```diff
@@ -23,6 +23,9 @@ trait BeerusApi {
     #[method(name = "hello_world")]
     async fn hello_world(&self) -> Result<String>;
 
+    #[method(name = "l2_to_l1_messages")]
```

Comment:
please change `l2_to_l1_messages` -> `starknet_l2_to_l1_messages` to maintain the syntax of the other endpoints.

Other than that LGTM!